modin/config/envvars.py

```diff
@@ -196,6 +196,8 @@ class DoUseCalcite(EnvironmentVariable, type=bool):
     """
 
     varname = "MODIN_USE_CALCITE"
+    default = True
+    choices = ("True", "False")
```

Comment:
I think we don't need to define `choices` here, by default the allowed set of values is larger (more flexible solution)

Reply:
Ok, corrected.